prestoadmin/util/remote_config_util.py

```diff
@@ -81,7 +81,7 @@ def lookup_string_config(config_value, config_file, host, default=''):
 
 def lookup_in_config(config_key, config_file, host):
     with settings(hide('stdout', 'warnings', 'aborts')):
-        config_value = execute(run, 'grep %s= %s' % (config_key, config_file),
+        config_value = execute(sudo, 'grep %s= %s' % (config_key, config_file),
```

Comment:
I'm not sure it matters in practice either, but it does limit the permissions you need to give somebody.